src/js/tracks/text-track.js

```diff
@@ -336,7 +336,17 @@ class TextTrack extends Track {
    * @param {TextTrack~Cue} cue
    *        The cue to add to our internal list
    */
-  addCue(cue) {
+  addCue(_cue) {
+    let cue = _cue;
```

Comment:
I think that it's more a self documenting code change that would be worth doing, as `_cue` doesn't really have any significance to tell you what it actually is.